tests/testsuite/fixture.go

```diff
@@ -201,6 +212,93 @@ func EnsureKVMPresent() {
 	}
 }
 
+func deleteFakeNodes() {
+	virtClient := kubevirt.Client()
+
+	err := virtClient.CoreV1().Nodes().DeleteCollection(context.TODO(), metav1.DeleteOptions{}, metav1.ListOptions{LabelSelector: "type=kwok"})
+	Expect(err).NotTo(HaveOccurred(), "failed to delete fake nodes")
+}
+
+// setup fake nodes for KWOK performance test
+func createFakeNodes() {
+	virtClient := kubevirt.Client()
+
+	By("create fake Nodes")
+	createFakeNodesWithKwok(virtClient, nodeCount)
+
+	By("Get the list of nodes")
+	nodeList, err := virtClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: "type=kwok"})
+	Expect(err).NotTo(HaveOccurred(), "Failed to list fake nodes")
+	Expect(nodeList.Items).To(HaveLen(nodeCount))
+
+}
+
+func createFakeNodesWithKwok(virtClient kubecli.KubevirtClient, nodeCount int) {
+	for i := 1; i <= nodeCount; i++ {
+		nodeName := fmt.Sprintf("kwok-node-%d", i)
+		node := newFakeNode(nodeName)
+
+		_, err := virtClient.CoreV1().Nodes().Create(context.TODO(), node, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Failed to create node %s", nodeName))
+	}
+}
```

Comment:
nit: this can be dropped moving the for loop directly inside the `createFakeNodes` function